cmd/internal/opentelemetry/wrapperMux.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opentelemetry
+
+import (
+	"flag"
+	"fmt"
+	"github.com/google/cadvisor/version"
+	"go.opentelemetry.io/otel/api/core"
+	"go.opentelemetry.io/otel/api/correlation"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/key"
+	"go.opentelemetry.io/otel/api/propagation"
+	apiTrace "go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/exporters/otlp"
+	"go.opentelemetry.io/otel/plugin/othttp"
+	"go.opentelemetry.io/otel/sdk/trace"
+	"google.golang.org/grpc/credentials"
+	"k8s.io/klog/v2"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+var (
+	traceSampler      = flag.String("trace_sampler", "always", "choose a sampler 'never', 'always' or 'probability'")
+	traceSamplerParam = flag.Float64("trace_sampler_param", 0, "used for the 'probability' sampler")
+	traceServiceName  = flag.String("trace_name", "cadvisor", "use localEndpoint service name")
```

Comment:
I think hard-coding this as cadvisor is fine to start with.  We don't need a flag.

Reply:
I have at moment 14 cAdvisor installation, and would like to differ the ServiceName in Zipkin and Jaeger.

That the reason for `trace_name`.

Thx Marcel

Reply:
is it possible to add the host name or something else as a tag to differentiate?  Passing a different flag to each instance is probably not ideal.